huaweicloud/services/waf/resource_huaweicloud_waf_domain.go

```diff
@@ -192,129 +218,82 @@ func resourceWafDomainCreate(ctx context.Context, d *schema.ResourceData, meta i
 			EnterpriseProjectId: epsID,
 		}
 
-		log.Printf("[DEBUG] Bind Waf domain %s to policy %s", d.Id(), policyID)
 		_, err = policies.UpdateHosts(wafClient, policyID, updateHostsOpts).Extract()
 		if err != nil {
-			return diag.Errorf("error updating WAF Policy Hosts: %s", err)
+			return diag.Errorf("error updating WAF policy Hosts: %s", err)
 		}
 
 		// delete the policy that was auto-created by domain
 		err = policies.DeleteWithEpsID(wafClient, domain.PolicyId, epsID).ExtractErr()
 		if err != nil {
-			log.Printf("[WARN] error deleting WAF Policy %s: %s", domain.PolicyId, err)
+			diag.Errorf("[WARN] error deleting WAF policy %s: %s", domain.PolicyId, err)
```

Comment:
Keep the original logic.
`log.Printf("[WARN] error deleting WAF policy %s: %s", domain.PolicyId, err)`